ApptentiveKit.Android/ApptentiveFeedback/ApptentiveFeedback/ApptentiveFeedback.csproj

```diff
@@ -26,18 +26,18 @@
     <PackageReference Include="Xamarin.AndroidX.Fragment" Version="1.6.2.2" />
 	  <PackageReference Include="Xamarin.Google.Android.Play.Review" Version="2.0.1.7" />
 	  <PackageReference Include="Xamarin.Google.Android.Play.Review.Ktx" Version="2.0.1.7"/>
-    <PackageReference Include="ApptentiveMessageCenter" Version="6.8.0"/>
+    <!-- <PackageReference Include="ApptentiveMessageCenter" Version="6.8.0"/>

```

Comment:
Should this still be commented out?

Reply:
Probably not. I am going to uncomment. This is to ensure the locally built packages are used. Thanks for catching!